homeassistant/components/wiz/config_flow.py

```diff
@@ -43,10 +44,18 @@ async def async_step_user(self, user_input=None):
                 errors["base"] = "unknown"
             else:
                 await self.async_set_unique_id(mac)
-                self._abort_if_unique_id_configured()
+                self._abort_if_unique_id_configured(
+                    updates={CONF_HOST: user_input[CONF_HOST]}
+                )
+                bulb_type = bulbtype.bulb_type.value if bulbtype else "Unknown"
+                name = f"{DEFAULT_NAME} {bulb_type} {_short_mac(mac)}"
                 return self.async_create_entry(
-                    title=user_input[CONF_NAME], data=user_input
+                    title=name,
+                    data={**user_input, CONF_NAME: name},
```

Comment:
`CONF_NAME` is now a duplicate of the entry title, we could use that one instead?

```suggestion
                    data=user_input,
```

Reply:
One less place to edit is a good thing. Its something new to learn that the title influences the name, but having two places is not good.  I think this is a good pattern to use elsewhere as well.  I'll probably adopt it in `flux_led` sometime later as well since editing both is a PITA :)